gb-cpu/src/memory/area/rom/nombc.rs

```diff
@@ -0,0 +1,26 @@
+use crate::bus::Bus;
+use crate::error::Error;
+
+pub struct NoMbc {
+    data: Vec<u8>,
+}
+
+impl Bus<usize> for NoMbc {
+    type Item = u8;
+    type Result = Result<(), Error>;
+    type Data = u8;
+
+    fn get(&self, address: usize) -> u8 {
+        *self.data.get(address).unwrap()
+    }
+
+    fn set(&mut self, _address: usize, _data: u8) -> Self::Result {
+        Ok(())
```

Comment:
I'm sure it's not `OK` to write in a RomOnly (ROM: Read Only Memory)

Reply:
We should be sure about what happends if a romonly game tries to write un memory, but i'm sure it does nothing and that the execution continues, that's what i represented here

Reply:
I agree, that's better